bodhi/tests/server/services/test_updates.py

```diff
@@ -5545,7 +5545,7 @@ def test_meets_testing_requirements_since_karma_reset_critpath(self, *args):
 
```

Comment:
We should check that `date_testing` gets blanked just like `karma` is reset to zero, since this is the scope of this PR. Can you add an assert statement? I don't think we need a separate test for that, we can do this just here.

Reply:
I did the change and tried seeing if the failing CI tests are working. 

`pytest-3 --no-cov --disable-warnings devel/ci/integration/tests/test_bodhi_cli.py::TestUpdatesService::test_meets_testing_requirements_since_karma_reset_critpath`

It's throwing a file not found error.

Reply:
You can't run integration tests with pytest.
You need to use `bci` command from the vagrant bodhi shell. Use `bci --help` to list the available commands.